### ISSUE TITLE
Deleting a group chat only deletes the group chat on your side

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-iOS#393]
If someone 'Deletes' the group chat from their conversations, the group goes away for them (as expected). But, they remain on the members list for all other contacts in the group (Still shown in members list for group). 

Bug still applies even if they are the creator of the group.

**Comment by N-Pex** [comment]
@cstiens Well, I don't think it's a bug. They are still listed as a member and could (theoretically at least) join the room again. See for example: https://stackoverflow.com/questions/21200550/leave-group-chat-and-get-removed-from-member-list

**Comment by cstiens** [comment]
You're right. This isn't a bug. Though, we do need to have a conversation again about how the experience of deleting a group works for the rest of the group (ie removing them from the roster). I'll move this to a later sprint.
